x-pack/filebeat/module/mssql/log/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: log
+  description: Common log fields
+  type: group
+  fields:
+    - name: source
```

Comment:
We should call this field different then `source` as it conflicts with the definition here: https://www.elastic.co/guide/en/ecs/current/ecs-source.html in ECS. It does not conflict field wise but if possible I would prefer to not have 2 meanings of `source`.

Also looking at the values of this field, I think it should be `keyword`

Reply:
I called it `origin` now